CHANGELOG.md

```diff
@@ -67,6 +67,13 @@
 
 ## Unreleased
 
+### Fixes
+
+#### Core
+
+- Failing connections are now handled properly for external plugins.
```

Comment:
Let's describe the fix in terms of behavior as seen by the user. For example: `Fix issue where failed connections would result in an inconsistent state in external plugins and high CPU utilization`.

Reply:
thanks @gszr! Done.